api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -2001,7 +2060,7 @@ public void testCloneWorkspaceCdrVersion() {
     assertThat(workspace2.getCdrVersionId()).isEqualTo(cdrVersionId2);
   }
 
-  @Test(expected = BadRequestException.class)
+  @Test(expected = NumberFormatException.class)
```

Comment:
Changed the logic check: now tries to parse the CDR Version ID string as Long.  The new test `testCloneWorkspaceMissingCdrVersion` checks for a CDR Version ID which is a long which is not present in the DB